src/main/webapp/resources/js/components/samples/components/SampleAnalyses.jsx

```diff
@@ -1,43 +1,153 @@
 import React from "react";
-import { Input, Space, Table } from "antd";
+import { Col, Input, Row, Table, Typography } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchSampleAnalyses } from "../../../apis/samples/samples";
+import { setSampleAnalyses } from "../sampleAnalysesSlice";
+
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import {
+  IconCheckCircle,
+  IconClock,
+  IconCloseCircle,
+  IconLoading,
+} from "../../icons/Icons";
+import { blue6, green6, grey6, red6 } from "../../../styles/colors";
 
 const { Search } = Input;
+const { Paragraph } = Typography;
+const commonIconStyle = {
+  fontSize: "16px",
+};
+
 /**
  * React component to display sample analyses
  *
  * @returns {JSX.Element}
  * @constructor
  */
 export function SampleAnalyses() {
+  const [filteredSubmissions, setFilteredSubmissions] = React.useState(null);
+  const dispatch = useDispatch();
+
+  const { sample } = useSelector((state) => state.sampleReducer);
+
+  const { analyses, loading } = useSelector(
+    (state) => state.sampleAnalysesReducer
+  );
+
+  /*
+  On page load get the sample analyses
+   */
+  React.useEffect(() => {
+    fetchSampleAnalyses({ sampleId: sample.identifier })
+      .then((analysesList) => {
+        dispatch(setSampleAnalyses({ analyses: analysesList }));
+      })
+      .catch((error) => {});
+  }, []);
+
   // Columns for the table
   const columns = [
     {
-      title: i18n("SampleAnalyses.sampleId"),
-      key: "sample_id",
+      title: i18n("SampleAnalyses.analysisSubmissionName"),
+      dataIndex: "name",
+      key: "name",
+      render(name, data) {
+        return (
+          <Paragraph ellipsis={{ rows: 1 }}>
+            <a
+              className="t-analysis-name"
+              href={setBaseUrl(`analysis/${data.id}`)}
+              title={name}
+              target="_blank"
+            >
+              {name}
+            </a>
+          </Paragraph>
+        );
+      },
     },
     {
-      title: i18n("SampleAnalyses.sampleName"),
-      key: "sample_name",
+      title: i18n("SampleAnalyses.analysisType"),
+      dataIndex: "analysisType",
+      key: "analysisType",
     },
     {
-      title: i18n("SampleAnalyses.analysisSubmission"),
-      key: "analysis_name",
+      title: i18n("SampleAnalyses.createdDate"),
+      dataIndex: "createdDate",
+      key: "createdDate",
+      width: 230,
+      render: (date) => (
+        <span className="t-analysis-created-date">
+          {date ? formatInternationalizedDateTime(date) : ""}
+        </span>
+      ),
     },
     {
-      title: i18n("SampleAnalyses.analysisType"),
-      key: "analysis_type",
+      title: i18n("SampleAnalyses.analysisState"),
+      dataIndex: "state",
+      key: "analysisType",
+      render: (state) => (
+        <span className="t-analysis-state">
```

Comment:
Make this a component

Reply:
Created component in 77a12d1d1497c2e6253bb09c7427afef56539876